packages/server/src/server.js

```diff
@@ -167,7 +167,7 @@ export default class Server {
         prefix: false,
         handler: (req, res) => {
           const to = urlJoin(this.nuxt.options.router.base, req.url)
-          consola.info(`[Development] Redirecting from \`${req.url}\` to \`${to}\` (router.base specified).`)
+          consola.info(`[Development] Redirecting from \`${decodeURI(req.url)}\` to \`${decodeURI(to)}\` (router.base specified).`)
```

Comment:
I'm sure there's a reason - but is there any merit in normalising on the `req` object?

Reply:
Actually it is a bad-request error for any http server with non-encoded URI. It is mostly to ensure not happening if `req` is not real `req` :)